Northstar.Custom/mod.json

```diff
@@ -440,6 +440,19 @@
 			"Path": "sh_northstar_safe_io.gnut",
 			"RunOn": "CLIENT || SERVER || UI"
 		},
+		{
+			"Path": "_testing.nut",
+			"RunOn": "CLIENT || SERVER || UI",
+			"ClientCallback": {
+				"Before": "Testing_Init"
+			},
+			"ServerCallback": {
+				"Before": "Testing_Init"
+			},
+			"UICallback": {
+				"Before": "Testing_Init"
```

Comment:
I guess this essentially calls a `NOP` but that shouldn't be a performance concern ig ^^